### ISSUE TITLE
estimate_dispersion() (and maybe estimate_zeroinflation()?)

### ISSUE THREAD

**Issue by bwiernik** [easystats/modelbased#136]
For models with dispersion models, it would be nice if two things were possible:

1. Incorporation of modeled dispersion parameters into prediction intervals in `estimate_prediction()`. 
2. Estimate predicted dispersion and confidence intervals on the link or response scale. This could be an option in `estimate_response()` and related functions (cf. https://github.com/easystats/insight/issues/413) or a separate `estimate_dispersion()`. Like with `estimate_prediction()`, making this an alias of `estimate_response()` with default options might make sense.

Relatedly, options for estimating zero-inflation predictions would be great.

**Comment by bwiernik** [comment]
What do you think about API here? @DominiqueMakowski

**Comment by strengejacke** [comment]
We _could_ add this easily, simply by calling `estimate_means()` with the appropriate `type` argument. So these functions could simply be wrappers.

**Comment by strengejacke** [comment]
I just saw that my comment didn't make sense. Brenton was talking about incorporating uncertainty into intervals, I was talking about predictions based on different model components.

**Comment by strengejacke** [comment]
> does that make sense?

Yes, sounds good. We could still use `estimate_means()` internally to get predictions for the different parameters, which is usually changed via the `type` argument. See, e.g., #278 where the ZI probabilities are predicted.
